app/assets/stylesheets/comments.scss

```diff
@@ -28,19 +41,124 @@
       border-left: 3px solid $blue;
       padding-left: 3px;
     }
+
+    .publisher-textarea-wrapper {
+      background-color: $white;
+      border: 1px solid $border-dark-grey;
+      border-radius: 3px;
+
+      &.with-attachments .photodropzone-container {
+        border-top: 1px dashed $light-grey;
+      }
+
+      .upload-error {
+        background-color: $red;
+        border-top: 1px solid $white;
+        color: $white;
+        font-weight: bold;
+        text-align: center;
+      }
+
+      .photodropzone {
+        list-style: none;
+
+        .publisher-photo {
+          float: left;
+          height: 80px;
+          line-height: 80px;
+          list-style: none;
+          margin: 8px;
```

Comment:
- Nesting should be no greater than 3, but was 4
- Selector should have depth of applicability no greater than 3, but was 5